recipe/meta.yaml

```
@@ -26,18 +26,23 @@ build:
     - move svgpathtools\svgpathtools wildmeshing\parse_svg  # [win]
     - dir wildmeshing\parse_svg  # [win]
     # - del svgpathtools  # [win]
-    - set "CMAKE_GENERATOR=NMake Makefiles"  # [win]
+    - set "CMAKE_GENERATOR=Ninja"  # [win]
+    - unset "CMAKE_PLAT" # [win]
+    - unset "CMAKE_GENERATOR_PLATFORM" # [win]
```

`unset` is bash, in Windows .bat you use `set` to an empty value

```suggestion
    - set "CMAKE_PLAT="  # [win]
    - set "CMAKE_GENERATOR_PLATFORM="  # [win]
```

---

thanks, let's give this a try!